docker/Makefile

```diff
@@ -0,0 +1,9 @@
+start:
+	docker compose -p dev --env-file .env up -d --build 
+
+start-test:
+	docker compose -p test --env-file .env.test up -d --build
+
+stop:
+	docker compose -p dev down --rmi local -v --remove-orphans
+	docker compose -p test down --rmi local -v --remove-orphans
```

Comment:
```suggestion
clean:
	docker compose -p dev down --rmi local -v --remove-orphans
	docker compose -p test down --rmi local -v --remove-orphans
```

Reply:
can we create 2 new ones: ?

- `down` - will only get docker-compose down, but not clean everything.
- `down-test` - same thing of above, but for test docker


reason for this is that not always we want to clean the docker... sometimes we want to stop the node, but keep contracts and other transaction we might already have inside local node